### PR TITLE
#163519147 Implement like and dislike article functionality

### PR DESCRIPTION
**Pull request by filschristian**
#### What does this PR do?
Have API end-points for liking and disliking an article

#### Description of Task to be completed?
Have the following API end-point working:
`POST: api/v1/articles/:slug/like`
`POST: api/v1/articles/:slug/dislike`

#### How should this be manually tested?

- Pull this branch

- Run `npm run migrate:seed`

- Run `npm run test`

- Run `npm run dev`

- With postman send post requests to those API end-points with the slug of the article you want to like or dislike

#### Any background context you want to provide?
Authentication required for these API end-points.

#### What are the relevant pivotal tracker stories?
#163519147

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A

### CODE REVIEW

**Review by mcaleb808 — APPROVED** [review]

File: .eslintrc [hunk 1]
```diff
@@ -48,6 +48,8 @@
           "ClassDeclaration": true
         }
       }
-    ]
+    ],
+    "object-curly-newline": ["error", { "multiline": true, "minProperties": 4 }],
```
Comment (oesukam) [inline .eslintrc]: @filschristian  We need this Eslint rule for long statements' readability. Try to fix the issue with Daniel

**Review by oesukam — CHANGES_REQUESTED** [review]
We can not continue with disabling eslint rule, let us keep what we have been using so far.

File: .eslintrc (hunk 1 above)
Comment (filschristian) [reply .eslintrc]: this rule does not have anything related to long statements!

Comment (oesukam) [reply .eslintrc]: @filschristian What is the use of the rule that you added?

Comment (filschristian) [reply .eslintrc]: @oesukam this line is for object's curly brackets

Comment (oesukam) [reply .eslintrc]: @filschristian Ok. But I have seen that you have removed some rules

Comment (filschristian) [reply .eslintrc]: @oesukam So I removed the max-line rule as you suggested but I wanted to let you know that this line you commented on was not the right one. You should have commented on max-line.

Comment (oesukam) [reply .eslintrc]: @filschristian Ok. Thanks for the feedback

**Review by oesukam — APPROVED** [review]
The PR has been reviewed ann approved

**Pull request merged**